webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js

```diff
@@ -65,6 +65,37 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
     if (typeof MathJax !== "undefined") {
       MathJax.Hub.Queue([ "Typeset", MathJax.Hub ]);
     }
+
+    window.addEventListener(
+      "message",
+      e => {
+        let message = e.data;
+        if ( typeof message == "string" ) message = JSON.parse(e.data);
+        if ( message.subject == "lti.gradeChangeNotify" ) {
```

Comment:
I'd either just return if this is not lti.gradeChangeNotify, or, if the intention is to leave a pattern for more message types, put it in a switch statement. Also, use the triple === for type and content equivalence checking.